functions/airtable.js

```diff
@@ -42,6 +42,8 @@ const ITEM_DIRECTORY_TABLE = functions.config().airtable.item_directory_table;
 
 const FINANCE_TRANSACTIONS_TABLE = functions.config().airtable.finance_transactions_table;
 
+const VAX_SUPPORT_TABLE = functions.config().airtable.vax_support_table;
```

Comment:
needs to be added to the config.

one thing i noticed is that in the current config (from the test firebase account, which is maybe why) the intake table name under `"intake_table": "_test_intake",`, but other tables seem to match the name in the base? i just want to make sure it's mapped correctly.

also, we'll also have to make this table in the base before running?

Reply:
yeah, at one point we didn't have a staging base and instead had test tables in the main base 😩 need to update that one